rotkehlchen/serialization/deserialize.py

```diff
@@ -543,7 +543,7 @@ def deserialize_ethereum_transaction(
             timestamp = deserialize_timestamp(data['timeStamp'])
 
         from_address = deserialize_ethereum_address(data['from'])
-        to_address = deserialize_ethereum_address(data['to']) if data['to'] != '' else None
+        to_address = deserialize_ethereum_address(data['to']) if (data['to'] != '' and data['to'] is not None) else None  # noqa: E501
```

Comment:
This is way too big a condition for an one liner at this point. Please break it up.

Reply:
Sure!